Fix warnings in Features apps

Please fix the following warnings

Warning: D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.WPF\Views\ActionsView.xaml(30,50): warning CS0108: 'ActionsView.Name' hides inherited member 'FrameworkElement.Name'. Use the new keyword if hiding was intended. [D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.WPF\Features.WPF.csproj]
Warning: D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.WPF\Views\NavigationSourceView.xaml(15,51): warning CS0108: 'NavigationSourceView.IsEnabled' hides inherited member 'UIElement.IsEnabled'. Use the new keyword if hiding was intended. [D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.WPF\Features.WPF.csproj]
Warning: D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.WPF\Views\NavigationTargetView.xaml(15,52): warning CS0108: 'NavigationTargetView.IsEnabled' hides inherited member 'UIElement.IsEnabled'. Use the new keyword if hiding was intended. [D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.WPF\Features.WPF.csproj]

Warning: D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.NetFive\Views\ActionsView.xaml(30,50): warning CS0108: 'ActionsView.Name' hides inherited member 'FrameworkElement.Name'. Use the new keyword if hiding was intended. [D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.NetFive\Features.DotNet_10rouhkz_wpftmp.csproj]
Warning: D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.NetFive\Views\NavigationSourceView.xaml(15,51): warning CS0108: 'NavigationSourceView.IsEnabled' hides inherited member 'UIElement.IsEnabled'. Use the new keyword if hiding was intended. [D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.NetFive\Features.DotNet_10rouhkz_wpftmp.csproj]
Warning: D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.NetFive\Views\NavigationTargetView.xaml(15,52): warning CS0108: 'NavigationTargetView.IsEnabled' hides inherited member 'UIElement.IsEnabled'. Use the new keyword if hiding was intended. [D:\a\Caliburn.Micro\Caliburn.Micro\samples\features\Features.NetFive\Features.DotNet_10rouhkz_wpftmp.csproj]

--- a/samples/features/Features.CrossPlatform.Shared/ViewModels/ActionsViewModel.cs
+++ b/samples/features/Features.CrossPlatform.Shared/ViewModels/ActionsViewModel.cs
@@ -12,18 +12,18 @@ namespace Features.CrossPlatform.ViewModels
 
         public void SimpleSayHello() => Output = "Hello from Caliburn.Micro";
 
-        public void SayHello(string name) => Output = $"Hello {name}";
+        public void SayHello(string username) => Output = $"Hello {username}";
 
-        public bool CanSayHello(string name) => !String.IsNullOrEmpty(name);
+        public bool CanSayHello(string username) => !String.IsNullOrEmpty(username);
 
-        public Task SayGoodbyeAsync(string name)
+        public Task SayGoodbyeAsync(string username)
         {
-            Output = $"Goodbye {name}";
+            Output = $"Goodbye {username}";
 
             return TaskHelper.FromResult(true);
         }
-        
-        public bool CanSayGoodbye(string name) => !String.IsNullOrEmpty(name);
+
+        public bool CanSayGoodbye(string username) => !String.IsNullOrEmpty(username);
 
         public string Output
         {

--- a/samples/features/Features.CrossPlatform.Shared/ViewModels/CoroutineViewModel.cs
+++ b/samples/features/Features.CrossPlatform.Shared/ViewModels/CoroutineViewModel.cs
@@ -1,6 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
+﻿using System.Collections.Generic;
 using Caliburn.Micro;
 using Features.CrossPlatform.Results;
 
@@ -17,7 +15,7 @@ namespace Features.CrossPlatform.ViewModels
 
             yield return new BusyResult(false);
 #else
-            yield return new VisualStateResult("Loading");
+            yield return new VisualStateResult("ShowLoading");
 
             yield return TaskHelper.Delay(2000).AsResult();
 

--- a/samples/features/Features.CrossPlatform.Shared/ViewModels/NavigationSourceViewModel.cs
+++ b/samples/features/Features.CrossPlatform.Shared/ViewModels/NavigationSourceViewModel.cs
@@ -9,32 +9,32 @@ namespace Features.CrossPlatform.ViewModels
 {
     public class NavigationSourceViewModel : Screen
     {
-        private readonly INavigationService navigationService;
-        private string text;
-        private bool isEnabled;
+        private readonly INavigationService _navigationService;
+        private string _text;
+        private bool _isNavigationEnabled;
 
         public NavigationSourceViewModel(INavigationService navigationService)
         {
-            this.navigationService = navigationService;
+            this._navigationService = navigationService;
         }
 
         public string Text
         {
-            get { return text; }
-            set { Set(ref text, value); }
+            get { return _text; }
+            set { Set(ref _text, value); }
         }
 
-        public bool IsEnabled
+        public bool IsNavigationEnabled
         {
-            get { return isEnabled; }
-            set { Set(ref isEnabled, value); }
+            get { return _isNavigationEnabled; }
+            set { Set(ref _isNavigationEnabled, value); }
         }
 
         public void Navigate()
         {
-            navigationService.For<NavigationTargetViewModel>()
+            _navigationService.For<NavigationTargetViewModel>()
                 .WithParam(v => v.Text, Text)
-                .WithParam(v => v.IsEnabled, IsEnabled)
+                .WithParam(v => v.IsNavigationEnabled, IsNavigationEnabled)
                 .Navigate();
         }
     }

--- a/samples/features/Features.CrossPlatform.Shared/ViewModels/NavigationTargetViewModel.cs
+++ b/samples/features/Features.CrossPlatform.Shared/ViewModels/NavigationTargetViewModel.cs
@@ -1,5 +1,4 @@
-﻿using System;
-using Caliburn.Micro;
+﻿using Caliburn.Micro;
 
 namespace Features.CrossPlatform.ViewModels
 {
@@ -14,7 +13,7 @@ namespace Features.CrossPlatform.ViewModels
             set { Set(ref text, value); }
         }
 
-        public bool IsEnabled
+        public bool IsNavigationEnabled
         {
             get { return isEnabled; }
             set { Set(ref isEnabled, value); }

--- a/samples/features/Features.NetFive/Views/ActionsView.xaml
+++ b/samples/features/Features.NetFive/Views/ActionsView.xaml
@@ -26,7 +26,7 @@
                 </TextBlock>
 
                 <TextBlock Text="Name"/>
-                <TextBox x:Name="Name" Margin="0,10,0,0" HorizontalAlignment="Stretch"/>
+                <TextBox x:Name="UserName" Margin="0,10,0,0" HorizontalAlignment="Stretch"/>
 
                 <Button x:Name="Clear" Content="Clear" Style="{StaticResource ActionButtonStyle}"/>
                 <Button x:Name="SimpleSayHello" Content="Simple Say Hello" Style="{StaticResource ActionButtonStyle}"/>
@@ -40,7 +40,7 @@
                     </i:Interaction.Triggers>
                 </Button>
                 <Button x:Name="SayHello" Content="Say Hello (with parameter)" Style="{StaticResource ActionButtonStyle}"/>
-                <Button cm:Message.Attach="SayHello(Name)" Content="Say Hello (with parameter and Message.Attach)" Style="{StaticResource ActionButtonStyle}"/>
+                <Button cm:Message.Attach="SayHello(UserName)" Content="Say Hello (with parameter and Message.Attach)" Style="{StaticResource ActionButtonStyle}"/>
                 <Button x:Name="SayGoodbye" Content="Say Goodbye (async method)" Style="{StaticResource ActionButtonStyle}"/>
             </StackPanel>
         </ScrollViewer>

--- a/samples/features/Features.NetFive/Views/CoroutineView.xaml
+++ b/samples/features/Features.NetFive/Views/CoroutineView.xaml
@@ -16,7 +16,7 @@
 
     <VisualStateManager.VisualStateGroups>
         <VisualStateGroup x:Name="LoadingStateGroup">
-            <VisualState x:Name="Loading">
+            <VisualState x:Name="ShowLoading">
                 <Storyboard>
                     <ObjectAnimationUsingKeyFrames Storyboard.TargetName="LoadingBar" Storyboard.TargetProperty="Visibility">
                         <DiscreteObjectKeyFrame KeyTime="0" Value="{x:Static Visibility.Visible}"/>

--- a/samples/features/Features.NetFive/Views/NavigationSourceView.xaml
+++ b/samples/features/Features.NetFive/Views/NavigationSourceView.xaml
@@ -11,7 +11,7 @@
     <Grid>
         <StackPanel Margin="20">
             <TextBox x:Name="Text" />
-            <CheckBox x:Name="IsEnabled" Content="Is Enabled" Margin="0,10" />
+            <CheckBox x:Name="IsNavigationEnabled" Content="Is Enabled" Margin="0,10" />
             <Button x:Name="Navigate" Content="Navigate" />
         </StackPanel>
     </Grid>

--- a/samples/features/Features.NetFive/Views/NavigationTargetView.xaml
+++ b/samples/features/Features.NetFive/Views/NavigationTargetView.xaml
@@ -11,7 +11,7 @@
     <Grid>
         <StackPanel Margin="20">
             <TextBlock x:Name="Text" />
-            <TextBlock x:Name="IsEnabled" />
+            <TextBlock x:Name="IsNavigationEnabled" />
         </StackPanel>
     </Grid>
 </Page>

--- a/samples/features/Features.UWP/Views/ActionsView.xaml
+++ b/samples/features/Features.UWP/Views/ActionsView.xaml
@@ -25,7 +25,7 @@
                 <Run Text="{Binding Output}"/>
             </TextBlock>
             
-            <TextBox Header="Name" x:Name="Name" Margin="0,10,0,0" Width="600" HorizontalAlignment="Left"/>
+            <TextBox Header="Name" x:Name="UserName" Margin="0,10,0,0" Width="600" HorizontalAlignment="Left"/>
 
             <Button x:Name="Clear" Content="Clear" Style="{StaticResource ActionButtonStyle}"/>
             <Button x:Name="SimpleSayHello" Content="Simple Say Hello" Style="{StaticResource ActionButtonStyle}"/>
@@ -39,7 +39,7 @@
                 </i:Interaction.Behaviors>
             </Button>
             <Button x:Name="SayHello" Content="Say Hello (with parameter)" Style="{StaticResource ActionButtonStyle}"/>
-            <Button cm:Message.Attach="SayHello(Name)" Content="Say Hello (with parameter and Message.Attach)" Style="{StaticResource ActionButtonStyle}"/>
+            <Button cm:Message.Attach="SayHello(UserName)" Content="Say Hello (with parameter and Message.Attach)" Style="{StaticResource ActionButtonStyle}"/>
             <Button x:Name="SayGoodbye" Content="Say Goodbye (async method)" Style="{StaticResource ActionButtonStyle}"/>
         </StackPanel>
     </StackPanel>

--- a/samples/features/Features.UWP/Views/CoroutineView.xaml
+++ b/samples/features/Features.UWP/Views/CoroutineView.xaml
@@ -17,7 +17,7 @@
         
         <VisualStateManager.VisualStateGroups>
             <VisualStateGroup x:Name="LoadingStateGroup">
-                <VisualState x:Name="Loading">
+                <VisualState x:Name="ShowLoading">
                     <VisualState.Setters>
                         <Setter Target="LoadingRing.IsActive" Value="True"/>
                     </VisualState.Setters>

--- a/samples/features/Features.UWP/Views/MenuView.xaml
+++ b/samples/features/Features.UWP/Views/MenuView.xaml
@@ -6,17 +6,18 @@
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:cm="using:Caliburn.Micro"
     mc:Ignorable="d">
-
-    <StackPanel Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-        <TextBlock Text="features" Style="{StaticResource SubheaderTextBlockStyle}" Margin="40,10,40,0"/>
-        <ListView x:Name="Features" Padding="40,20" SelectionMode="None" IsItemClickEnabled="True"
+    <ScrollViewer>
+        <StackPanel Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+            <TextBlock Text="features" Style="{StaticResource SubheaderTextBlockStyle}" Margin="40,10,40,0"/>
+            <ListView x:Name="Features" Padding="40,20" SelectionMode="None" IsItemClickEnabled="True"
                   cm:Message.Attach="[Event ItemClick] = [ShowFeature($clickedItem)]" >
-            <ListView.ItemContainerStyle>
-                <Style TargetType="ListViewItem">
-                    <Setter Property="Margin" Value="0,0,0,10"/>
-                    <Setter Property="Padding" Value="10"/>
-                </Style>
-           </ListView.ItemContainerStyle>
-        </ListView>
-    </StackPanel>
+                <ListView.ItemContainerStyle>
+                    <Style TargetType="ListViewItem">
+                        <Setter Property="Margin" Value="0,0,0,10"/>
+                        <Setter Property="Padding" Value="10"/>
+                    </Style>
+                </ListView.ItemContainerStyle>
+            </ListView>
+        </StackPanel>
+    </ScrollViewer>
 </Page>

--- a/samples/features/Features.UWP/Views/NavigationSourceView.xaml
+++ b/samples/features/Features.UWP/Views/NavigationSourceView.xaml
@@ -11,7 +11,7 @@
         <TextBlock Text="navigation" Style="{StaticResource SubheaderTextBlockStyle}" Margin="40,10,40,0"/>
         <StackPanel Margin="40,20">
             <TextBox Header="Text" x:Name="Text" />
-            <CheckBox x:Name="IsEnabled" Content="Is Enabled" />
+            <CheckBox x:Name="IsNavigationEnabled" Content="Is Enabled" />
             <Button x:Name="Navigate" Content="Navigate" />
         </StackPanel>
     </StackPanel>

--- a/samples/features/Features.UWP/Views/NavigationTargetView.xaml
+++ b/samples/features/Features.UWP/Views/NavigationTargetView.xaml
@@ -11,7 +11,7 @@
         <TextBlock Text="navigation" Style="{StaticResource SubheaderTextBlockStyle}" Margin="40,10,40,0"/>
         <StackPanel Margin="40,20">
             <TextBlock x:Name="Text" Style="{StaticResource BodyTextBlockStyle}" />
-            <TextBlock x:Name="IsEnabled" Style="{StaticResource BodyTextBlockStyle}" />
+            <TextBlock x:Name="IsNavigationEnabled" Style="{StaticResource BodyTextBlockStyle}" />
         </StackPanel>
     </StackPanel>
 </Page>

--- a/samples/features/Features.WPF/Views/ActionsView.xaml
+++ b/samples/features/Features.WPF/Views/ActionsView.xaml
@@ -26,7 +26,7 @@
                 </TextBlock>
 
                 <TextBlock Text="Name"/>
-                <TextBox x:Name="Name" Margin="0,10,0,0" HorizontalAlignment="Stretch"/>
+                <TextBox x:Name="UserName" Margin="0,10,0,0" HorizontalAlignment="Stretch"/>
 
                 <Button x:Name="Clear" Content="Clear" Style="{StaticResource ActionButtonStyle}"/>
                 <Button x:Name="SimpleSayHello" Content="Simple Say Hello" Style="{StaticResource ActionButtonStyle}"/>
@@ -40,7 +40,7 @@
                     </i:Interaction.Triggers>
                 </Button>
                 <Button x:Name="SayHello" Content="Say Hello (with parameter)" Style="{StaticResource ActionButtonStyle}"/>
-                <Button cm:Message.Attach="SayHello(Name)" Content="Say Hello (with parameter and Message.Attach)" Style="{StaticResource ActionButtonStyle}"/>
+                <Button cm:Message.Attach="SayHello(UserName)" Content="Say Hello (with parameter and Message.Attach)" Style="{StaticResource ActionButtonStyle}"/>
                 <Button x:Name="SayGoodbye" Content="Say Goodbye (async method)" Style="{StaticResource ActionButtonStyle}"/>
             </StackPanel>
         </ScrollViewer>

--- a/samples/features/Features.WPF/Views/CoroutineView.xaml
+++ b/samples/features/Features.WPF/Views/CoroutineView.xaml
@@ -16,7 +16,7 @@
 
     <VisualStateManager.VisualStateGroups>
         <VisualStateGroup x:Name="LoadingStateGroup">
-            <VisualState x:Name="Loading">
+            <VisualState x:Name="ShowLoading">
                 <Storyboard>
                     <ObjectAnimationUsingKeyFrames Storyboard.TargetName="LoadingBar" Storyboard.TargetProperty="Visibility">
                         <DiscreteObjectKeyFrame KeyTime="0" Value="{x:Static Visibility.Visible}"/>

--- a/samples/features/Features.WPF/Views/NavigationSourceView.xaml
+++ b/samples/features/Features.WPF/Views/NavigationSourceView.xaml
@@ -11,7 +11,7 @@
     <Grid>
         <StackPanel Margin="20">
             <TextBox x:Name="Text" />
-            <CheckBox x:Name="IsEnabled" Content="Is Enabled" Margin="0,10" />
+            <CheckBox x:Name="IsNavigationEnabled" Content="Is Enabled" Margin="0,10" />
             <Button x:Name="Navigate" Content="Navigate" />
         </StackPanel>
     </Grid>

--- a/samples/features/Features.WPF/Views/NavigationTargetView.xaml
+++ b/samples/features/Features.WPF/Views/NavigationTargetView.xaml
@@ -11,7 +11,7 @@
     <Grid>
         <StackPanel Margin="20">
             <TextBlock x:Name="Text" />
-            <TextBlock x:Name="IsEnabled" />
+            <TextBlock x:Name="IsNavigationEnabled" />
         </StackPanel>
     </Grid>
 </Page>

